.changeset/cyan-lions-float.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-graphiql': patch
+---
+
+Finalized composability API migration, now exporting the plugin as `graphiqlPlugin`.
```

Comment:
a silly thing but, i am glad it's `graphiql` instead of `graphiQl`, since I get dictionary squigglies for `graphi` that I don't want to add to the personal dictionary :P